static-site/src/components/nav-bar/index.tsx

```diff
@@ -119,8 +119,10 @@ const NavBar = ({ minified }: {
 
   return (
     <NavContainer>
-      <Logo />
-      <LogoType />
+      {Router.pathname != "/" && <>
+        <Logo />
+        <LogoType />
+      </>}
```

Comment:
_non-blocking_ 

Since there's discussion in this PR regarding the responsiveness on mobile, I wanted to note that the "LOGIN" button is lost in mobile view because of the presence of the logo (looking at the [/pathogens page](https://nextstrain-s-victorlin--xbfmig.herokuapp.com/pathogens)). 

![Screenshot 2024-07-29 at 1 13 16 PM](https://github.com/user-attachments/assets/356fea52-a2f0-493d-8314-dd212fc2cb8d)

A nice future improvement would be to collapse the nav links behind button at certain breakpoints, but for now maybe just hide the logo based on breakpoint?

Reply:
That's already the behavior without this PR, so I've created a separate issue: #962

Reply:
> That's already the behavior without this PR

Ah, guess I just haven't looked at the mobile view in a while!